### DISCUSSION TITLE
Wiki Contributions

### DISCUSSION THREAD

**Discussion by robswc** [robswc/tradingview-webhooks-bot#35] [Ideas]
Here will be the discussion related to wiki contributions

**Comment by robswc** [comment]
So, I think the best first step would be if everyone interested in helping could check out this branch:
https://github.com/robswc/tradingview-webhooks-bot/tree/2021.1

And use the guide in the readme.md and wiki, to see if the instructions for setting things up are correct/aren't missing anything.

From there, if suggestions for the following could be posted:

1. Any parts of the guide that are confusing.
2. Any parts of the guide that are redundant.
3. Any parts of the guide that should be moved around/restructured.

I've just been buried in it for so long, I'm not sure what needs expanding on, is unclear, etc.

Thanks!

**Reply by ApopheniaPays** [reply]
Appreciate your responses, and glad to see this project is live. Speaking for myself, my interest is definitely high, but I'm kind of buried, too, in things that are less interesting but technically higher priority. 

This is actually very good timing though, as I very recently completed a trading algorithm in Pine script that seems to work well, but profitview, which I've been using to automate my crypto trades from TradingView alerts, has been seriously unreliable, and, doesn't interface with my stock broker at all, it's just for crypto exchanges. So this is well-timed. I was seriously hoping to find something that would be a little more plug-n-play and a little less reinventing-the-wheel, but if I have to contribute work to make it happen, then so be it, this is probably a good effort to throw in with. 

So, I kind of envision maybe replacing profitview with this to capture my tradingview alerts, and then using (hopefully) fairly complete projects like tda-api and ccxt (just to guess at two) to actually execute the trades. Yeah, that's the ticket. 

As to the wiki, I've written and edited documentation before, so I could definitely see contributing there, if I can get familiar enough with this to. 

So, let me review as I have time and post back. If it gets to be a while, I invite the dev or any other interested party to rattle my cage, either by replying here, on twitter at this same username, or email githubberish[at]apopheniapays[doot]com.
